Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -1231,13 +1281,17 @@ protected class ClassWriter : IClassWriter {
         case BinaryExpr.ResolvedOpcode.Mod:
           staticCallString = $"{DafnyRuntimeModule}.euclidian_modulus"; break;
 
-        case BinaryExpr.ResolvedOpcode.Lt:
```

Comment:
I don't see this moving somewhere else, is it just not supported now?

Reply:
Is supported by `SinglePassCompiler`.